commission/views/commission_settlement.xml

```diff
@@ -0,0 +1,174 @@
+                        widget="monetary"
+                        options="{'currency_field': 'currency_id'}"
+                    />
+                </group>
+            </form>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_settlement_line_tree">
+        <field name="name">Settlement lines</field>
+        <field name="model">commission.settlement.line</field>
+        <field name="arch" type="xml">
+            <tree create="false">
+                <field name="date" />
+                <field name="agent_id" />
+                <field name="commission_id" />
+                <field name="settled_amount" sum="Settled total" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_settlement_line_search" model="ir.ui.view">
+        <field name="name">commission.settlement.line.search</field>
+        <field name="model">commission.settlement.line</field>
+        <field name="arch" type="xml">
+            <search string="Settlement lines search">
+                <field name="agent_id" />
+                <field name="date" />
+                <field name="commission_id" />
+                <filter
+                    string="This year"
+                    name="thisyear"
+                    domain="['|', ('date', '=', False), '&amp;',('date','&lt;=', time.strftime('%%Y-12-31')),('date','&gt;=',time.strftime('%%Y-01-01'))]"
+                />
+                <separator />
+                <group expand="1" string="Group By">
+                    <filter
+                        string="Agent"
+                        name="group_agent"
+                        context="{'group_by': 'agent_id'}"
+                    />
+                    <filter
+                        string="Date month"
+                        name="group_date"
+                        context="{'group_by': 'date:month'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="view_settlement_line_graph" model="ir.ui.view">
+        <field name="name">commission.settlement.line.graph</field>
+        <field name="model">commission.settlement.line</field>
+        <field name="arch" type="xml">
+            <graph string="Settlement analysis" sample="1">
+                <field name="agent_id" />
+                <field name="date" interval="month" operator="+" />
+                <field name="settled_amount" operator="+" />
+            </graph>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="action_settle_commission">
+        <field name="name">Settlements</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">commission.settlement</field>
+        <field name="binding_view_types">form,tree</field>
+    </record>
+    <menuitem
+        name="Settlements"
+        id="menu_settle_commissions"
+        parent="menu_commissions_management"
```

Comment:
> Thought about having "Commission Management" node under Invoicing, but that I think will orphan the node when the module is used with the enterprise account_accountant.

I tried to put the new menu "Commission Management" under invoicing app. But just like you mentioned when the module is used with enterprise account_accountant, two apps are appeared in home screen (Invoicing and Accounting). There is only "Commission Management" menu under Invoicing app.

Reply:
> I suggest that some of the commission menu items, such as Agents, Settlements and Create commission invoices be moved under Invoicing > Vendors.

I think Agents should be under configuration menu because that menu is for defining agent(My POV).